numpy/lib/function_base.py

```diff
@@ -4733,12 +4733,12 @@ def append(arr, values, axis=None):
     return concatenate((arr, values), axis=axis)
 
 
-def _digitize_dispatcher(x, bins, right=None):
+def _digitize_dispatcher(x, bins, right=None, edge=None):
     return (x, bins)
 
 
 @array_function_dispatch(_digitize_dispatcher)
-def digitize(x, bins, right=False):
+def digitize(x, bins, right=False, edge=False):
```

Comment:
I think we should have made this keyword-only

Reply:
+1

Reply:
Hmm, again, I think with style it's the same as `right` as it is so I'm not sure what the argument is to make it different. If they were to both be keyword only then that would make sense.

Reply:
What's different is that `right` was added before keyqord-only arguments were possible, and changing it now would break callers.